### PR TITLE
Displaying HappyHours

### PR DESCRIPTION
**Pull request by NikBorn**
@joshuajhun, @nicktu12, @lolakoala
## What's this PR do?

adds a card and container component  to display happy hour information from firebase. 

## Where should the reviewer start?
In HappyHourContainer and HappyHourCard

## Any background context you want to provide?
this is iteration 0 for getting the data to display, given more time I will have the data displaying on the actual location card and not on a separate map.

## Screenshots (if appropriate)
screen shot 2017-11-07 at 7 48 54 pm

## What gif best describes this PR or how it makes you feel?
source

### CODE REVIEW

File: src/Components/HappyHourCard/HappyHourCard.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        </td>
+        <td className='happy-data'>
+          {happyHours[day][2]}
+        </td>
+      </tr>
+    );
+  });
+
+  return (
+    <div className='happy-hour-card'>
```
Comment (lolakoala) [inline src/Components/HappyHourCard/HappyHourCard.js]: I commend your bravery is working with a table. I fucking hate tables.

**Comment by bbp5280** [comment]
Looks good. Nice work.

File: src/Components/HappyHourContainer/HappyHourContainer.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import HappyHourCard from '../HappyHourCard/HappyHourCard.js';
+import LoadingScreen from '../LoadingScreen/LoadingScreen';
+
+
+
```
Comment (lolakoala) [inline src/Components/HappyHourContainer/HappyHourContainer.js]: Nice little lean component here. Good job separating out responsibilities.

File: src/Utils/mockData.js [hunk 3]
```diff
@@ -436,4 +436,42 @@ export const mockUser = {
...
   expirationTime: 1509582786564
-};
\ No newline at end of file
+};
+
+export const mockHappyHourInfo = [
```
Comment (lolakoala) [inline src/Utils/mockData.js]: Omg, I love some mock data!

**Pull request merged**